src/popcorn-list-movies/test/PopcornListMovies.test.js

```diff
@@ -44,11 +48,23 @@ describe("PopcornListMovies", () => {
 
   it("should return nothing when there are no movies in the homepage", async () => {
     const element = await scopedFixture(
-      html `<popcorn-list-movies></popcorn-list-movies>`
+      html`<popcorn-list-movies></popcorn-list-movies>`
     );
 
-    const popcornListMovies = element.shadowRoot.querySelector('[data-testid="popcorn-list-movies"]');
+    const popcornListMovies = element.shadowRoot.querySelector(
+      '[data-testid="popcorn-list-movies"]'
+    );
 
     expect(popcornListMovies).to.be.null;
   });
-});
\ No newline at end of file
+
+  it("should return a poster from assets folder when the image is N/A", async () => {

```

Comment:
You could rephrase this as: "should return a placeholder when the provided poster is invalid"

Reply:
Also it would be great if you could create another "it" testing the opposite scenario.